index.html

```diff
@@ -32,23 +32,157 @@ <h3>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+
+      <article>
+        <a href="#"><img src="assets/work_2.png" alt="Preview of this project"></a>
+        <h2><a href="#">Multi-Post Stories</a></h2>
+        <div class="info">
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
+        <h3 class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+        <ul class="tags">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+
+      <article>
+        <a href="#"><img src="assets/work_3.png" alt="Preview of this project"></a>
+        <h2><a href="#">Tonic</a></h2>
+        <div class="info"> 
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
+        <h3 class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+        <ul class="tags">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+
+      <article>
+        <a href="#"><img src="assets/work_4.png" alt="Preview of this project"></a>
+        <h2><a href="#">Multi-Post Stories</a></h2>
+        <div class="info">
+          <span><a href="#">CANOPY</a></span>
+          <div class="dot"></div>
+          <span><a href="#">Back End Dev</a></span>
+          <div class="dot"></div>
+          <span><a href="#">2015</a></span>
+        </div>
+        <h3 class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</h3>
+        <ul class="tags">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javaScript</a></li>
+        </ul>
+        <button type="button" class="btn-more">See Project</button>
+      </article>
+    </div>
+
+    <section class="about-myself">
+      <h2>
+        About<br>
+        Myself
+      </h2>
+
+      <h3>
+        I&rsquo;m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don&rsquo;t hesitate to contact me.
+      </h3>
+
+      <span class="emphasize"><a href="#">LET&rsquo;S CONNECT</a></span>
+
```

Comment:
- [ ] I would suggest you to use `hx` tag as it is a title not the link

Reply:
A previous reviewer on pull request #1 explicitly told me this should be a link that will redirect to the form in the future but I've changed the wrapping element to be `hx` instead of `span`.